tests/cnf/core/network/metallb/tests/common.go

```diff
@@ -240,6 +240,15 @@ func setupBgpAdvertisement(
 	Expect(err).ToNot(HaveOccurred(), "Failed to create BGPAdvertisement")
 }
 
+func updateBgpAdvertisement(
```

Comment:
pls rename the func. It updates NodeSelector

Reply:
renamed